moped-editor/src/layouts/DashboardLayout/Footer.js

```diff
@@ -1,42 +1,42 @@
 import React, { useEffect, useState } from "react";
 import { Typography } from "@mui/material";
-import makeStyles from '@mui/styles/makeStyles';
+import makeStyles from "@mui/styles/makeStyles";
 import ExternalLink from "../../components/ExternalLink";
 import { get } from "lodash";
 
 var pckg = require("../../../package.json");
 
-
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
   root: {
     paddingTop: theme.spacing(1),
     paddingLeft: theme.spacing(4),
     paddingBottom: theme.spacing(2),
   },
 }));
 
+/**
+ * Async function that retrieves most recent tag from an API call to Github "tag" endpoint
+ */
+const getMostRecentGithubTags = async () => {
+  const githubTagsApiEndpoint =
+    "https://api.github.com/repos/cityofaustin/atd-moped/tags";
+  const response = await fetch(githubTagsApiEndpoint);
+  const data = await response.json();
+  if (get(data, 0)) {
+    return data[0].name;
+  } else {
+    return `v${pckg.version}`;
+  }
```

Comment:
my brain is moving slow right now... so we link to the latest release version, regardless of whether or not that release tag matches the package version? i don't mean to open this can of worms...

one more question, should we handle when the `response` is not `ok`? iirc we have an unhandled error when the github call fails.

Reply:
worms welcome! i have also wondered about whether our good release habits could eliminate this api call altogether. am i thinking about this right? we need two things:

- current full version number (aka `2.19.2`) which we can get from `package.json`
- a link to the latest tags which we can reach with https://github.com/cityofaustin/atd-moped/releases/latest  <-- this was a TIL

Reply:
oh i didnt realize that link redirect existed! I want to get rid of the api call 

I scope creeped these worms and I have no regrets

Reply:
Same - it was a nice surprise! So cool to have this feature covered in new and less GH API-dependent way!! 🚀